scripts/exgdas_efsoi_update.sh

```diff
@@ -0,0 +1,413 @@
+   sattypes_rad(42)= 'sndrd2_g14',    dsis(42)= 'sndrD2_g14',
+   sattypes_rad(43)= 'sndrd3_g14',    dsis(43)= 'sndrD3_g14',
+   sattypes_rad(44)= 'sndrd4_g14',    dsis(44)= 'sndrD4_g14',
+   sattypes_rad(45)= 'sndrd1_g15',    dsis(45)= 'sndrD1_g15',
+   sattypes_rad(46)= 'sndrd2_g15',    dsis(46)= 'sndrD2_g15',
+   sattypes_rad(47)= 'sndrd3_g15',    dsis(47)= 'sndrD3_g15',
+   sattypes_rad(48)= 'sndrd4_g15',    dsis(48)= 'sndrD4_g15',
+   sattypes_rad(49)= 'iasi_metop-a',  dsis(49)= 'iasi_metop-a',
+   sattypes_rad(50)= 'seviri_m08',    dsis(50)= 'seviri_m08',
+   sattypes_rad(51)= 'seviri_m09',    dsis(51)= 'seviri_m09',
+   sattypes_rad(52)= 'seviri_m10',    dsis(52)= 'seviri_m10',
+   sattypes_rad(53)= 'seviri_m11',    dsis(53)= 'seviri_m11',
+   sattypes_rad(54)= 'amsua_metop-b', dsis(54)= 'amsua_metop-b',
+   sattypes_rad(55)= 'hirs4_metop-b', dsis(55)= 'hirs4_metop-b',
+   sattypes_rad(56)= 'mhs_metop-b',   dsis(56)= 'mhs_metop-b',
+   sattypes_rad(57)= 'iasi_metop-b',  dsis(57)= 'iasi_metop-b',
+   sattypes_rad(58)= 'avhrr_metop-b', dsis(58)= 'avhrr3_metop-b',
+   sattypes_rad(59)= 'atms_npp',      dsis(59)= 'atms_npp',
+   sattypes_rad(60)= 'atms_n20',      dsis(60)= 'atms_n20',
+   sattypes_rad(61)= 'cris_npp',      dsis(61)= 'cris_npp',
+   sattypes_rad(62)= 'cris-fsr_npp',  dsis(62)= 'cris-fsr_npp',
+   sattypes_rad(63)= 'cris-fsr_n20',  dsis(63)= 'cris-fsr_n20',
+   sattypes_rad(64)= 'gmi_gpm',       dsis(64)= 'gmi_gpm',
+   sattypes_rad(65)= 'saphir_meghat', dsis(65)= 'saphir_meghat',
+   sattypes_rad(66)= 'amsua_metop-c', dsis(66)= 'amsua_metop-c',
+   sattypes_rad(67)= 'mhs_metop-c',   dsis(67)= 'mhs_metop-c',
+   sattypes_rad(68)= 'ahi_himawari8', dsis(68)= 'ahi_himawari8',
+   sattypes_rad(69)= 'abi_g16',       dsis(69)= 'abi_g16',
+   sattypes_rad(70)= 'abi_g17',       dsis(70)= 'abi_g17',
+   $SATOBS_ENKF
+/
+&ozobs_enkf
+   sattypes_oz(1) = 'sbuv2_n16',
+   sattypes_oz(2) = 'sbuv2_n17',
+   sattypes_oz(3) = 'sbuv2_n18',
+   sattypes_oz(4) = 'sbuv2_n19',
+   sattypes_oz(5) = 'omi_aura',
+   sattypes_oz(6) = 'gome_metop-a',
+   sattypes_oz(7) = 'gome_metop-b',
+   sattypes_oz(8) = 'mls30_aura',
+   sattypes_oz(9) = 'ompsnp_npp',
+   sattypes_oz(10) = 'ompstc8_npp',
+   $OZOBS_ENKF
+/
+EOFnml
+
+################################################################################
+# Run enkf update
+
+export OMP_NUM_THREADS=$NTHREADS_ENKF
+export pgm=$ENKFEXEC
+. prep_step
+
+$NCP $ENKFEXEC $DATA
+$APRUN_ENKF ${DATA}/$(basename $ENKFEXEC) 1>stdout 2>stderr
+rc=$?
+
+export ERR=$rc
+export err=$ERR
+$ERRSCRIPT || exit 2
+
+# save for EFSOI task (still needed?)
+$NCP $COMOUT_ANL_ENS/$GBIASe $COMOUT_ANL_ENSFSOI
+
+# Cat runtime output files.
+cat stdout stderr > $COMOUT_ANL_ENSFSOI/$ENKFSTAT
+
+################################################################################
+#  Postprocessing
+######## AFE remove after testing
+#cp -r $DATA /scratch1/NCEPDEV/stmp4/Andrew.Eichmann/efsoi
```

Comment:
remove commented lines and blocks that are/were for debugging.

Reply:
removed